README.md

```diff
@@ -1,4 +1,4 @@
-
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/atlp-rwanda/alpha-ec-be/tree/develop.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/atlp-rwanda/alpha-ec-be/tree/develop)
```

Comment:
Why did you change this link

Reply:
i added this link it is new that's why it is here